pkg/controller/volume/persistentvolume/scheduler_binder_test.go

```diff
@@ -461,21 +462,8 @@ func (env *testEnv) validateBind(
 	t *testing.T,
 	name string,
 	pod *v1.Pod,
-	expectedPVs []*v1.PersistentVolume,
 	expectedAPIPVs []*v1.PersistentVolume) {
 
-	// Check pv cache
-	pvCache := env.internalBinder.pvCache
-	for _, pv := range expectedPVs {
-		cachedPV, err := pvCache.GetPV(pv.Name)
-		if err != nil {
-			t.Errorf("Test %q failed: GetPV %q returned error: %v", name, pv.Name, err)
-		}
-		if !reflect.DeepEqual(cachedPV, pv) {
```

Comment:
I think after bind, we still want to make sure that the cache reflects the binding decision. So that a future predicate will use the updated bindings.

Is it the resource version that's causing problems?  Instead of doing a DeepEqual, can we just check the information we really care about, ie PVC selectedNode, and PV.ClaimRef?

Reply:
I see, we can compare but ignore resource version. PR is updated.